### PR TITLE
Aws dev

### PR DESCRIPTION
**Pull request by vippatel90**

### CODE REVIEW

File: src/app/app.service.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    (<AWS.CognitoIdentityCredentials>AWS.config.credentials).get(function () {
+      // Credentials will be available when this function is called.
+      const accessKeyId = AWS.config.credentials.accessKeyId;
+      const secretAccessKey = AWS.config.credentials.secretAccessKey;
+      const sessionToken = AWS.config.credentials.sessionToken;
+    });
+  }
+
+  detectLabels(imageData) {
+    // AWS.region = "us-east-2";
```
Comment (vippatel90) [inline src/app/app.service.ts:26]: this is for detect labels. same we we can add detect face in this service. 
Service is returning ES2015 Promise

File: src/app/app.service.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import * as AWS from 'aws-sdk';
+
+@Injectable()
+export default class AppService {
+  constructor() {
+    this.anonLog();
+  }
+
+  anonLog() {
+    // Configure the credentials provider to use your identity pool
```
Comment (vippatel90) [inline src/app/app.service.ts:11]: this is to configure AWS creds pool

File: src/tsconfig.app.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
     "outDir": "../out-tsc/app",
     "baseUrl": "./",
     "module": "es2015",
-    "types": []
+    "types": ["node"]
```
Comment (vippatel90) [inline src/tsconfig.app.json:7]: required for aws- sdk as per aws-sdk guideline for typescript

File: src/styles.css [hunk 4]
```diff
@@ -1 +1,2 @@
 /* You can add global styles to this file, and also import other style files */
+@import '~@angular/material/prebuilt-themes/deeppurple-amber.css';
```
Comment (vippatel90) [inline src/styles.css:2]: required for material progress bar css

**Review by Shishir-Tiwari — APPROVED** [review]

**Pull request merged**